steps/apply/spec.schema.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "properties": {
+    "os": {
+      "type": "object",
+      "description": "Additional packages and commands to run prior to terraform execution",
+      "properties": {
+        "packages": {
+          "type": "array",
+          "description": "list of apk packages to install at container execution time"
+        },
+        "commands": {
+          "type": "array",
+          "description": "List of shell commands to run after packages are installed but before Terrafrom executes"
+        }
+      }
+    },
+    "workspace": {
+      "type": "string",
+      "description": "The name of the Terraform workspace to run (defaults to 'default')"
+    },
+    "credentials": {
```

Comment:
This "older" syntax for accessing GCP is something like [this](https://github.com/relay-integrations/relay-gcp/blob/master/steps/gcp-step-cloud-run-deploy/spec.schema.json#L6), and probably worth noting that the contents have to be Base64-encoded